docs/README.md

```diff
@@ -11,96 +15,78 @@
   - [Dependencies](#getting-started)
   - [Development](#dependencies)
   - [Deployment](#development)
-- [Authors](#deployment)
-- [Acknowledgments](#authors)
-- [About Chingu](#acknowledgments)
-- [Contact Us](#about-chingu)
+- [Authors](#authors)
+- [Acknowledgments](#acknowledgements)
+- [About Chingu](#about-chingu)
+- [Contact Us]()
+
 
 ## Overview
 
-Chingu Voyage 48 is a six-week project where a team of seven individuals, including a scrum master and a product owner, developed a web application for learning about dinosaurs
+Chingu Voyage 48 is a six-week project where a team of 7 individuals, including a Scrum Master and a product owner, developed a single page application (SPA) for learning about Dinosaurs.
 
 ### The Project
 
-The Minimum Viable Product of this application ...
+The Minimum Viable Product of this SPA allows users to: 
+- View a comprehensive list of dinosaur names and images.
+- Easily find specific dinosaurs by name.
+- Understand the distribution of dinosaur diets with an informative chart.
+- Access detailed information about each dinosaur, including type, size, diet, era, species.
+
+#### Example Requirement Image
 
-#### Example Image
+<img width="1332" alt="image" src="https://github.com/chingu-voyages/v48-tier1-team-04/assets/161270515/cb66b3a5-8698-4748-91ab-6fff551c8db9">
 
-![Example daily task productivity tool](./assets/daily_productivity_sample_ui.png)
 
 This project provided an opportunity for the team to gain hands-on experience in web development and work with multi-dimensional datasets. It served as a practical application to hone their skills and delve into the complexities of data manipulation within a web-based environment.
 
 ### Screenshot
 
-![Screenshot of the application](./assets/screenshot.png)
+<img width="1682" alt="image" src="https://github.com/chingu-voyages/v48-tier1-team-04/assets/161270515/3e00d1e9-e0df-4051-b7d8-2e1423e784a5">
 
-### [Video Showcase](https://www.veed.io/view/30855dda-33a1-4c9b-88df-84e3ef15774d?panel=share)
+
+### [Video Showcase]
 
 ### Links
 
-- [Repository](https://github.com/chingu-voyages/v48-tier1-team-04)
-- [Documentation](https://chingu-voyages.github.io/v48-tier1-team-04/)
-- [Live Site URL](https://v48-tier1-team-04.netlify.app/)
-- [Development URL](https://v48-tier1-team-04-dev.netlify.app/)
+- [Repository](https://github.com/chingu-voyages/v48-tier1-team-04/)
+- [Documentation](https://github.com/chingu-voyages/v48-tier1-team-04/tree/development/docs)
+- [Live Site URL](https://v48-tier1-team-04-dev.netlify.app/)
+- [Development URL](https://chingu-raptors.netlify.app/)
 
 ### Built with
 
+- [Figma]()
 - [HTML: HyperText Markup Language](https://developer.mozilla.org/en-US/docs/Web/HTML)
 - [CSS: Cascading Style Sheets](https://developer.mozilla.org/en-US/docs/Web/CSS)
```

Comment:
We could say SASS here which stands for [Syntactically Awesome CSS](https://sass-lang.com/)